src/Components/TemporaryDrawer.js

```diff
@@ -0,0 +1,305 @@
+            <IconButton
+              onClick={() => {
+                setSelectedLandmarks(natureParks);
+                setLandmarksChange(true);
+                console.log(landmarksChange);
+              }}
+            >
+              <SailingIcon sx={{ margin: "20px" }} />
+              Nature Parks
+            </IconButton>
+          </ListItemIcon>
+          <ListItemIcon>
+            <IconButton
+              onClick={() => {
+                setSelectedLandmarks(politicalLandmarks);
+                setLandmarksChange(true);
+                console.log(landmarksChange);
+              }}
+            >
+              <TakeoutDiningIcon sx={{ margin: "20px" }} />
+              Political Landmarks
+            </IconButton>
+          </ListItemIcon>
+        </List>
+        {/* <List>
+          {['Singapore Flyer', 'Sentosa Island', 'Chinatown Singapore'].map(
+            (text, index) => (
+              <ListItemButton key={text}>
+                <ListItemIcon>
+                  {text === 'Singapore Flyer' ? (
+                    <IconButton
+                      variant="contained"
+                      onClick={() => {
+                        const message =
+                          "Singapore Flyer's history in 1 sentence";
+                        sendMessage(message);
+                      }}
+                      sx={{ width: '50%', marginBottom: '20px' }}
+                    >
+                      <AttractionsIcon />
+                    </IconButton>
+                  ) : text === 'Sentosa Island' ? (
+                    <IconButton
+                      variant="contained"
+                      onClick={() => {
+                        const message =
+                          "Sentosa Island's history in 1 sentence";
+                        sendMessage(message);
+                      }}
+                      sx={{ width: '50%', marginBottom: '20px' }}
+                    >
+                      <SailingIcon />
+                    </IconButton>
+                  ) : text === 'Chinatown Singapore' ? (
+                    <IconButton
+                      variant="contained"
+                      onClick={() => {
+                        const message =
+                          "Chinatown Singapore's history in 1 sentence";
+                        sendMessage(message);
+                      }}
+                      sx={{ width: '50%', marginBottom: '20px' }}
+                    >
+                      <TakeoutDiningIcon />
+                    </IconButton>
+                  ) : null}
+                </ListItemIcon>
+                <ListItemText
+                  primary={text}
+                  onClick={() => {
+                    const message = `${text}'s history in 1 sentence`;
+                    sendMessage(message);
+                  }}
+                />
+              </ListItemButton>
+            ),
+          )}
+        </List> */}
```

Comment:
Holy moly :D Why not just mark and press backspace?

Reply:
ctrl + shift + d also removes a full line (or marked lines) in vscode
or
cmd + shift + d